src/Widget/Error.php

```diff
@@ -4,27 +4,52 @@
 
 namespace Yiisoft\Form\Widget;
 
+use Yiisoft\Form\FormModelInterface;
 use Yiisoft\Form\Helper\HtmlForm;
-use Yiisoft\Form\Widget\Attribute\ModelAttributes;
 use Yiisoft\Html\Tag\CustomTag;
 use Yiisoft\Widget\Widget;
 
 /**
  * The Error widget displays an error message.
+ *
+ * @psalm-suppress MissingConstructor
  */
 final class Error extends Widget
 {
-    use ModelAttributes;
-
+    private array $attributes = [];
```

Comment:
This is confusing. I'd name it `tagAttributes`.

Reply:
https://github.com/terabytesoftw/yii-form/blob/883d98d085c527eb8387f1f2db126029ba0d4c0b/src/Widget/Label.php#L39 those are the global attributes of the widget.

Reply:
done.